src/PrestaShopBundle/Resources/views/Admin/AdvancedParameters/system_information.html.twig

```diff
@@ -265,16 +226,63 @@
         </div>
     </div>
 </div>
-{% if notHostMode %}
+{% if system.notHostMode %}
     <div class="card">
         <div class="card-block">
-            <h3 class="card-heading">
-                <i class="material-icons">info_outline</i> {{ 'List of changed files'|trans({}, 'Admin.Advparameters.Feature') }}
+            <h3 class="card-header">
+                <i class="material-icons">info_outline</i> {{ 'List of changed files'|trans }}
             </h3>
             <div class="card-text" id="changedFiles">
                 <i class="material-icons">loop</i> {{ 'Checking files...'|trans({}, 'Admin.Advparameters.Notification') }}
             </div>
         </div>
     </div>
 {% endif %}
+{% if system.notHostMode %}
+    <script>
+        $(document).ready(function()
+        {
+            var translations = {
+                missing: '{{ "Missing files"|trans({}, "Admin.Advparameters.Notification") }}',
```

Comment:
I think it's preferable to use
```js
missing: {{ "Missing files"|trans({}, "Admin.Advparameters.Notification")|json_encode }},
```
because it takes away the "escaping javascript quotes" problem.

Reply:
`json_encode` doesn't works the way you expect:
![json_encode](https://user-images.githubusercontent.com/1247388/30265013-1d1faf9a-96db-11e7-9670-e05eb5c43caa.png)

If I quote and "json_encode" this content, I have to clean my strings after to avoid multiples quotes:
![not_good](https://user-images.githubusercontent.com/1247388/30265074-47d4ce0a-96db-11e7-9e8b-6dc1e650a5e0.png)

Obviously, I don't want to do that: the content returned by server is server responsability, it's not up to javascript to clean content from server.

Reply:
All comments have been fixed.

Reply:
json_encode normally escapes things for javascript. The problem here is that the resulting json is being auto-escaped by twig, which results in invalid json.

This should be easily solved by adding `|raw`

```twig
"Missing files"|trans({}, "Admin.Advparameters.Notification")|json_encode|raw
```

Reply:
This is what does `js` escaping option I'm now using => https://github.com/PrestaShop/PrestaShop/pull/8254/files#diff-2283959d3c048970b5d7af6fa22ed662R245 :+1:

Reply:
Ok then